templates/aup_in_form.tpl.php

```diff
@@ -23,85 +23,45 @@
   $aupListEndpoint = $this->data['aupListEndpoint'];
 }
 
-$this->data['jquery'] = array('core' => true, 'ui' => true, 'css' => true);
-$this->data['head'] = header("Expires: Thu, 19 Nov 1981 08:52:00 GMT"). //Date in the past in order not to keep cache if someone goes back to that page
-header("Cache-Control: no-store, no-cache, must-revalidate"). //HTTP/1.1 
-'<style type="text/css">
-.aup_content{
-   cursor: pointer;
-}
-.aup_rows:nth-child(odd) {
-    background: #fafafa; 
+// Get Configuration and set the loader
+$globalConfig = SimpleSAML_Configuration::getInstance();
+$theme_use = $globalConfig->getString('theme.use', 'default');
```

Comment:
what happens in the case of the default theme? will there be an error or the loader will be missing? we should add a note to the README that the UpdateAUP filter expects a `loader` configuration option for blah blah

Reply:
In case of the default theme there will be no loader. Only in case of a custom theme it will check for the loader. If the key is not present then getValue will return null. As a result, no loader will be loaded.
Perhaps i could make the name of the loader configurable with default value then key `loader`